features/furniture/caterer-popup.feature.md

```diff
@@ -0,0 +1,42 @@
+# Feature: Furniture: Caterer Popup
```

Comment:
I'm wondering if "Caterer Popup" is a little bit too specific? Couldn't something like this just generally be an "order here" piece of furniture? What are the specific features that this needs that are no part of "select things form a menu and order them for delivery"?